network/p2p/gossip/gossip.go

```diff
@@ -323,9 +327,10 @@ type PushGossiper[T Gossipable] struct {
 }
 
 type BranchingFactor struct {
-	Validators    int
-	NonValidators int
-	Peers         int
+	StakePercentage float64
+	Validators      int
```

Comment:
nit: change to `UniformValidators` so it is clear that this is done differently rather than playing with `StakePercentage`?

Reply:
I decided to document these values rather than trying to re-name them.